voluncheer/templates/profiles/posttogallery.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block content %}
+{% if user.is_volunteer %}
+
+</div>
+<!-- Side Nav -->
```

Comment:
Wouldn't it make sense to have this in a template like like "base_volunteer.html" and then extend from there? Maybe not for this user story, but just thinking out loud on how to reduce repetition.

Reply:
Thing is there aren't allowed to have more than one base.html extensions. We can definitely change this, but I think it will be not worth the effort

Reply:
So we can't have profile_base.html extend base.html, then posttogallery.html extend profile_base.html? I agree it may not be worth it at this point.